editor/plugins/node_3d_editor_plugin.h

```diff
@@ -833,9 +832,9 @@ class Node3DEditor : public VBoxContainer {
 	ToolMode get_tool_mode() const { return tool_mode; }
 	bool are_local_coords_enabled() const { return tool_option_button[Node3DEditor::TOOL_OPT_LOCAL_COORDS]->is_pressed(); }
 	bool is_snap_enabled() const { return snap_enabled ^ snap_key_enabled; }
-	float get_translate_snap() const;
-	float get_rotate_snap() const;
-	float get_scale_snap() const;
+	double get_translate_snap() const;
+	double get_rotate_snap() const;
+	double get_scale_snap() const;
```

Comment:
Why is it double not real_t?

Reply:
These methods get data from strings, so there's no need to lose precision early.